bucket/sioyek.json

```
@@ -29,6 +29,7 @@
             "}"
         ]
     },
+    "env_add_path": ".",
```

```suggestion
    "bin": "sioyek.exe",
```
As long as the shimgen has no issues, there's no reason to pollute path here.